CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+2.1.6 Release notes (2017-03-02)
+=============================================================
+* Updated to Realm 2.4.3
```

Comment:
It might be worth calling out that this fixes an issue with token expiration for Realm's opened from Realm Mobile Platform.